main.go

```diff
@@ -87,15 +117,8 @@ func main() {
 
 	log.Printf("* xcodebuild_version: %s (%s)", xcodebuildVersion.Version, xcodebuildVersion.BuildVersion)
 
-	// xcpretty version
-	if cfgs.OutputTool == "xcpretty" {
-		xcprettyVersion, err := GetXcprettyVersion()
-		if err != nil {
-			failf("Failed to get the xcpretty version! Error: %s", err)
-		} else {
-			log.Printf("* xcpretty_version: %s", xcprettyVersion)
-		}
-	}
+	// xcpretty
+	cfgs.OutputTool = s.ensureXCPretty()
```

Comment:
Can we merge this line with the other part of xcpretty handling at line 155? I was looking at the code in between and it does not use this output tool value.

Reply:
My idea was that this is the `InstallDeps` phase, while what we have in line 155 is the `Run` phase of the step.